### PR TITLE
fix: increase gas limit

### PR DESCRIPTION
**Pull request by kylehoang92**

### CODE REVIEW

File: src/constant/generic.ts [hunk 1]
```diff
@@ -8,9 +8,9 @@ import { Blockchain, BlockchainV2, EVMChain as EVMChainV2 } from "@carbon-sdk/ut
 export const DEFAULT_FEE_DENOM = "swth";
 
 // 0.0000001 SWTH
-// 10 SWTH sats
+// 15 SWTH sats
 export const DEFAULT_GAS_PRICE = new BigNumber(10);
-export const DEFAULT_GAS = new BigNumber(10000000);
+export const DEFAULT_GAS = new BigNumber(15000000);
```
Comment (sarah-thong) [inline src/constant/generic.ts:13]: Hi, don't think this is enough. Cos the implementation is such that the SDK will take the default value from the [API call](https://api.carbon.network/carbon/fee/v1/gas_costs?pagination.limit=10000) (which is `10000000`), and only if the API call returns nothing, then we'll use `15000000`.
image

Think you may need to find all submitted txs on Demex, and then override the gas cost for each of them.

**Comment by sarah-thong** [comment]
### UI Comments
- [ ] Staking txs gas is still 20m, should be 100m.
  Screenshot 2024-09-24 120141

**Comment by kylehoang92** [comment]
https://github.com/Switcheo/switcheo-disco/pull/3537

File: src/constant/generic.ts [hunk 2]
```diff
@@ -8,9 +8,9 @@ import { Blockchain, BlockchainV2, EVMChain as EVMChainV2 } from "@carbon-sdk/ut
 export const DEFAULT_FEE_DENOM = "swth";
 
 // 0.0000001 SWTH
-// 10 SWTH sats
+// 100 SWTH sats
```
Comment (randy75828) [inline src/constant/generic.ts:11]: 10^8 isnt it just 1 SWTH

**Review by randy75828 — APPROVED** [review]

File: src/constant/generic.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import { Blockchain, BlockchainV2, EVMChain as EVMChainV2 } from "@carbon-sdk/ut
 export const DEFAULT_FEE_DENOM = "swth";
 
 // 0.0000001 SWTH
-// 100 SWTH sats
```
Comment (sarah-thong) [inline src/constant/generic.ts]: I'm not quite sure how sats work. Maybe can remove this comment since we're not sure if this is correct.

**Review by sarah-thong — APPROVED** [review]

**Pull request merged**